app/templates/users/add_user.hbs

```diff
@@ -0,0 +1,147 @@
+<nav class="tab-bar new-user-nav">
+ <section class="left-small" {{action "back"}}>
+    <i class="back_icon" aria-hidden="true">{{fa-icon 'angle-left'}}</i>
+    <div>{{t 'back'}}</div>
+  </section>
+  <section class="middle tab-bar-section">
+    <h1 class="title">{{t 'users.create_new_user'}}</h1>
+  </section>
+</nav>
+
+<section class="main-section add-user-form new-user-section ui ">
+  {{#async-validatable-form onSubmit=(action 'saveUser') as |form|}}
+  <div class="row">
+    <div class="small-5 columns item-label">
+      {{t 'users.image'}}
+    </div>
+    <div class="small-7 columns">
+      <a {{action "triggerUpload"}} class="{{unless isReady 'disabled'}}">
+        <span class="image-box">
+          {{#if newUploadedImage}}
+          <img src={{newUploadedImage.thumbImageUrl}} class="thumb" />
+          {{else}}
+          <i aria-hidden="true">{{fa-icon 'camera'}}</i><br />
+          {{t 'users.add_image'}}
+          {{/if}}
+        </span>
+      </a>
+      {{cloudinary-upload ready="uploadReady" progress="uploadProgress"
+       always="uploadComplete" done="uploadSuccess" submit="uploadStart"}}
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="small-5 columns item-label">
+      {{t 'users.title'}}
+    </div>
+    <div class="small-7 columns field-container title-select">
+      {{select-list
+      content=titles
+      on-change=(action (mut selectedTitle))
+      selectedValue=selectedTitle
+      value= selectedTitle
+      selectArrow=true
+      idNames='title' }}
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="small-5 columns item-label">
+      {{t 'users.first_name'}}
+    </div>
+    <div class="small-7 columns field-container">
+      {{#form-control}}
+      {{input type='text' value=firstName  required='true'}}
+      <div class='input-error'>{{t 'organisation.user.validation_error.first_name'}}</div>
+      {{/form-control}}
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="small-5 columns item-label">
+      {{t 'users.last_name'}}
+    </div>
+    <div class="small-7 columns field-container">
+      {{#form-control}}
+      {{input type='text' value=lastName  required='true'}}
+      <div class='input-error'>{{t 'organisation.user.validation_error.family_name'}}</div>
+      {{/form-control}}
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="small-5 columns item-label">
+      {{t 'users.email'}}
+    </div>
+    <div class="small-7 columns field-container">
+      {{#form-control}}
+        {{input value=email name="email" id="email" type="email" required=setEmailorMobile }}
```

Comment:
As i mentioned above, any person reading this would get confused
`required=setEmailorMobile`
always important to have a proper naming convention : )